Updated installation steps

btkcodedev opened the pull request.

btkcodedev committed:
Updated installation steps

btkcodedev committed:
Patch 2

Added contribution steps
Used correct terminologies

btkcodedev committed:
Patch 3

Changed some of contribution section,
Cleansed the steps further, by removing unwanted details

btkcodedev committed:
patch 4

BreadGenie commented on README.md:
```diff
@@ -26,29 +26,29 @@ Technique of Encryption
   <h4>LSB based Image steganography:</h4>
```
the heading here is not according to the hierarchy. Can you maybe change that?

BreadGenie replied:
And the ``` ## Contributing ``` at line 99

btkcodedev replied:
Done @BreadGenie

btkcodedev committed:
Resolved heading hierarchy and cleansed the errors

Made every heading with h3 tag
Cleansed the Installation steps from wrong tags and included spaces.

btkcodedev committed:
Merge pull request #2 from btkcodedev/btkcodedev-patch-5

Resolved heading hierarchy and cleansed the errors

BreadGenie commented:
Thanks for the contribution @btkcodedev!
